test_problems/pureFluidTest/testPureWater.cpp

```diff
@@ -24,30 +24,28 @@ int main()
     double pres;
     try {
 
-        XML_Node* xc = get_XML_File("liquidvapor.xml");
-        XML_Node* const xs = xc->findNameID("phase", "water");
-        ThermoPhase* water_tp = newPhase(*xs);
-        PureFluidPhase* w = dynamic_cast <PureFluidPhase*>(water_tp);
+        PureFluidPhase w;
+        w.initThermoFile(std::string("liquidvapor.yaml"), std::string("water"));
```

Comment:
I don't think the explicit call to the `std::string` constructor is necessary, e.g. you should be able to write just `w.initThermoFile("liquidvapor.yaml", "water");`